index.js

```diff
@@ -0,0 +1,56 @@
+const movesUp = "("
+const movesDown = ")"
+
+
+// First exercise --------------------
+const countSantasMoves = function(inputValue, santasDirection) {
+  const checkSantasMoves = new RegExp(`\\${santasDirection}`, 'g');
+  
+  if (inputValue == null) {
```

Comment:
Isso aqui podia ser a primeira linha dessa função, pq se `inputValue` for `null` ele não precisa executar aquela Regex acima.